README.md

```diff
@@ -39,40 +36,34 @@ jobs:
 
 ### `enforce-sqg`
 
-If set to `true`, forces to complete execution successfully even if the failures conditions (SQG criteria) you have set are met.
+If set to `true`, the task with return a failure when security quality gates (SQG) criteria have failed.
+If set to `false`, the action reports SQG failures scenarios without enforcing them (i.e. give a grace period to development teams before you start breaking builds).
 
-This parameter can be useful if you want to detect SQG failures scenarios without enforcing them (i.e. give a grace period to development teams before you start breaking builds).
-
-Default is `false`.
-
-Implementation: get the SQG status from audit report - 
+Default is `false`.  
 
 ### `log-level`
 
-Sets the level of details in the logs, one of: `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`. 
+Sets the level of details in the action logs, one of: `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`. 
 Default is `INFO`.
 
 ### `data-enrich`
 
-Enrichs the OpenAPI file leveraging the default data dictionary. For each property with a standard format (such as uuid or date-time), patterns and constraints will be added to the OpenAPI file before it is audited. 
+Enrichs the OpenAPI file leveraging 42Crunch default data dictionary. For each property with a standard format (such as uuid or date-time), patterns and constraints will be added to the OpenAPI file before it is audited.
```

Comment:
"Enriches..."

I'd also get rid of the will here, it's not really necessary: "...constraints are added..."